apps/nextjs-website/react-components/types/Hero/Hero.types.ts

```diff
@@ -14,10 +14,16 @@ export interface StoreButtonsProps {
   readonly hrefApple?: string;
 }
 
+export interface ChipProps {
+  readonly label: string;
+  readonly targetId: string;
```

Comment:
```suggestion
  readonly targetID: string;
```
For consistency with other components